server/routes/rpaFramework.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const router = express.Router();
+
+//Require controller modules
+const commandsController = require('../controllers/rpaFrameworkCommandsController');
+
+/// RPAFRAMEWORKCOMMANDS ROUTES ///
+
+router.get(
+  '/commands/get-available-applications',
+  commandsController.getAvailableApplications
+);
+
+router.get(
+  '/commands/get-available-tasks-for-application',
+  commandsController.getAvailableTasksForApplications
+);
+
+router.get('/commands/get-vars-for-task', commandsController.getVarsForTask);
```

Comment:
Shouldn't styling be constistent?

Reply:
Good point!
@kej-jay please do not use rhetorical questions as they are sometimes harder to understand.

Reply:
I don't really get what specifically you are referring to with "styling"

Reply:
Okay Lukas just told me what you meant with "styling". This styling is done automatically by my vs-code formatting on save. I use "Prettier - Code formatter" which is the must commonly used formatter on vs code.